docs/sessions/sessions.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "How it works?"
+sidebar_label: "How it works?"
+---
+
+GoBlitz uses default session-based authentication written in Go. Controller uses models to interact with the database. Middleware `Authentication` function validates the session and checks if the user is authorized or not.
+
+The `Authentication` function is also used to make an apiKey based authentication for the Status Page.
+
+## Signup
+
+On the signup the controller renders data from `loginFormData` struct and compares with the data in database. 
+
+The controller uses model to check the user existence by getting a boolean value. If the user exists, the controller returns an error message. If the user doesn't exist, the controller creates a record in database about the user and sets the session.
+
+## Signin
+
+On the signin the controller renders data from `loginFormData` struct and compares with the data in database. 
+
+The controller uses model to check for the user existance and compares the password input with the encrypted record in database.
+
+If the user exists and the password is correct, the controller sets the session. If the user doesn't exist or the password is incorrect, the controller returns an error message.
```

Comment:
I would use two words for the sign up and sign in, as it's a two-word verb phrase, or at least in the text sign-in, sign-up.

Reply:
You decided not to change it?